### PR TITLE
LPS-203953 Show the rowchecker if the user is able to download the file

### PR DESCRIPTION
**Pull request by robertoDiaz**

### CODE REVIEW

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp [hunk 1]
```diff
@@ -77,7 +77,7 @@ entriesChecker.setRememberCheckBoxStateURLRegex("^(?!.*" + liferayPortletRespons
 							latestFileVersion = fileEntry.getLatestFileVersion();
 						}
 
-						if ((dlSearchContainer.getRowChecker() == null) && (DLFileEntryPermission.contains(permissionChecker, fileEntry, ActionKeys.DELETE) || DLFileEntryPermission.contains(permissionChecker, fileEntry, ActionKeys.UPDATE))) {
+						if ((dlSearchContainer.getRowChecker() == null) && (DLFileEntryPermission.contains(permissionChecker, fileEntry, ActionKeys.DELETE) || DLFileEntryPermission.contains(permissionChecker, fileEntry, ActionKeys.DOWNLOAD) || DLFileEntryPermission.contains(permissionChecker, fileEntry, ActionKeys.UPDATE))) {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp:80]: this should be done on
line https://github.com/liferay/liferay-portal/blob/72447fc3708065aa8e158956431ecc036bc63139/modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp#L395 too?

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp (hunk 1 above)
Comment (robertoDiaz) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp:80]: Folders do not have the dowload permission

Comment (robertoDiaz) [reply modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/search_resources.jsp:80]: Hey @AliciaGarciaGarcia 

I hadn't noticed that this permission was added to Folders lately.

I'll add it.

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by Jeremy-Ch** [comment]
ci:test:stable

**Comment by Jeremy-Ch** [comment]
ci:forward:force

**Pull request closed without merging**